base/operators.jl

```diff
@@ -873,17 +873,25 @@ const ÷ = div
     mod1(x, y)
 
 Modulus after flooring division, returning a value `r` such that `mod(r, y) == mod(x, y)`
-in the range ``(0, y]`` for positive `y` and in the range ``[y,0)`` for negative `y`.
+in the range ``(0, y]`` for positive `y`, and in the range ``[y,0)`` for negative `y`.
```

Comment:
there is no list, so it doesn't have an extra comma before the conjunction
```suggestion
in the range ``(0, y]`` for positive `y` and in the range ``[y,0)`` for negative `y`.
```